src/data/categories.js

```diff
@@ -0,0 +1,2 @@
+const categories = ['Action', 'Adventure', 'Biography', 'History', 'Horror', 'Kids', 'Learning', 'Sci-Fi'];
+export default categories;
```

Comment:
You need to add `'ALL'` to the options [see [requirements](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/bookstore/milestone4/AllReturnsAllCategories.png)]